[2025-07-04T08:57:18.205+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aggregate_query_to_temp_logs.run_aggregate_query manual__2025-07-04T08:57:16.812078+00:00 [queued]>
[2025-07-04T08:57:18.210+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aggregate_query_to_temp_logs.run_aggregate_query manual__2025-07-04T08:57:16.812078+00:00 [queued]>
[2025-07-04T08:57:18.211+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-04T08:57:18.218+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_aggregate_query> on 2025-07-04 08:57:16.812078+00:00
[2025-07-04T08:57:18.223+0000] {standard_task_runner.py:60} INFO - Started process 14518 to run task
[2025-07-04T08:57:18.225+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'aggregate_query_to_temp_logs', 'run_aggregate_query', 'manual__2025-07-04T08:57:16.812078+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/aggregate_query_dag.py', '--cfg-path', '/tmp/tmpzk97gib3']
[2025-07-04T08:57:18.226+0000] {standard_task_runner.py:88} INFO - Job 63: Subtask run_aggregate_query
[2025-07-04T08:57:18.270+0000] {task_command.py:423} INFO - Running <TaskInstance: aggregate_query_to_temp_logs.run_aggregate_query manual__2025-07-04T08:57:16.812078+00:00 [running]> on host 0dc0f72acdaa
[2025-07-04T08:57:18.364+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='aggregate_query_to_temp_logs' AIRFLOW_CTX_TASK_ID='run_aggregate_query' AIRFLOW_CTX_EXECUTION_DATE='2025-07-04T08:57:16.812078+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-04T08:57:16.812078+00:00'
[2025-07-04T08:57:18.375+0000] {base.py:83} INFO - Using connection ID 'snowflake_dev_conn' for task execution.
[2025-07-04T08:57:18.375+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.8.18, Platform: Linux-6.2.0-26-generic-x86_64-with-glibc2.34
[2025-07-04T08:57:18.376+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-07-04T08:57:19.237+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:221 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-07-04T08:57:19.237+0000] {cursor.py:1028} INFO - query: [SELECT DEPARTMENT, COUNT(*) AS TOTAL_EMPLOYEES, ROUND(AVG(SALARY), 2) AS AVG_SAL...]
[2025-07-04T08:57:19.526+0000] {cursor.py:1041} INFO - query execution done
[2025-07-04T08:57:19.528+0000] {cursor.py:1205} INFO - Number of results in first chunk: 5
[2025-07-04T08:57:19.538+0000] {connection.py:718} INFO - closed
[2025-07-04T08:57:19.646+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2025-07-04T08:57:19.756+0000] {aggregate_query_dag.py:15} INFO - Aggregated Results:
[2025-07-04T08:57:19.765+0000] {aggregate_query_dag.py:16} INFO -  DEPARTMENT  TOTAL_EMPLOYEES  AVG_SALARY  MAX_SALARY  MIN_SALARY
    Finance                5    83800.00     94000.0     60000.0
         HR                4    58500.00     61000.0     55000.0
Engineering                4    92750.00     96000.0     88000.0
      Sales                4    72000.00     75000.0     70000.0
  Marketing                3    65333.33     69000.0     63000.0
[2025-07-04T08:57:19.765+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-04T08:57:19.780+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=aggregate_query_to_temp_logs, task_id=run_aggregate_query, execution_date=20250704T085716, start_date=20250704T085718, end_date=20250704T085719
[2025-07-04T08:57:19.847+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-04T08:57:19.878+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
